IFC4x3/Properties/w/WithLightningRod_3pPNiRy4X9KvANfnRk92$0/DocProperty.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocProperty xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="WithLightningRod" UniqueId="f3657b1b-f048-4953-9297-a716ee242fc0" PrimaryDataType="IfcBoolean" />
```

Comment:
orphan property?

Reply:
@jmirtsch  there is a collection of properties in this request that are not in any of the PSets? could this be sorted semi automatically?